packer/wowza/provisioning.json

```diff
@@ -0,0 +1,28 @@
+{
+    "variables": {
+        "wowza_ami": "ami-91960ce2"
+    },
+    "builders": [{
+        "type": "amazon-ebs",
+        "region": "eu-west-1",
+        "source_ami": "{{user `wowza_ami`}}",
+        "instance_type": "c3.large",
```

Comment:
I actually think `c3.large` may be too much for packer. Packer will just spin up the box, run some commands, create an ami then terminate the box.

We'd specify `c3.xlarge` in the cloudformation that spins up the instance from this build ami.